_includes/footer.html

```diff
@@ -93,7 +93,7 @@ <h3>Get support</h3>
       <div class="usa-identifier__container">
           <div class="usa-identifier__logos">
               <a href="javascript:void(0);" class="usa-identifier__logo">
-                  <img class="usa-identifier__logo-img" src="{{ site.baseurl }}/img/gsa-logo.svg" alt="GSA logo" role="img">
+                  <img class="usa-identifier__logo-img" src="{{ site.baseurl }}/img/gsa-logo.svg" alt="GSA logo" role="img" width="48" height="48">
```

Comment:
Same here, this is a core USWDS component. If it makes a difference we should make the same enhancement to the component.